### ISSUE TITLE
Quick question: Is this different/better than your EnhanceIO_old depot? 

### ISSUE THREAD

**Issue by hradec** [elmystico/EnhanceIO#1]
Just wondering... I want to grab the latest code available, and your EnhanceIO_old seems to be the latest, but then I've found this one in your depot list. (the "_old" got me wondering). 

So, is this one stable and usable as well? should I grab this one?

**Comment by elmystico** [comment]
Nah, old one is old- maybe the repo got changed later. See on code changes-
the latest were on current, not _old.

_old is not usable

czw., 3.05.2018, 23:40 użytkownik Hradec <notifications@github.com> napisał:
